packages/grid/src/features/table-ui/components/InfiniteList.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from "react";
+import * as Infinite from "react-infinite";
+
+interface Props {
+  length: number | undefined;
+  threshold?: number;
+  elementHeight: number;
+}
+
+export const InfiniteList: React.FC<Props> = ({
+  length = 0,
+  threshold = 100,
+  children,
+  elementHeight
+}) => {
+  if (length < threshold) {
```

Comment:
why?

Reply:
Lazy rendering when scrolling can be annoying.
Only applying it if the number of items reaches a threshold.